packages/paste-website/src/pages/experiences/console/alarm-setup/index.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Alarm Setup
+description: Guidelines for the Alarm Setup experience in Console.
+slug: /experiences/console/alarm-setup
+---
+
+import {graphql} from 'gatsby';
+import {GatsbyImage} from 'gatsby-plugin-image';
+import {SidebarCategoryRoutes} from '../../../../constants';
+import {WatchThisSpace} from '../../../../components/empty-state/WatchThisSpace';
+
+export const pageQuery = graphql`
+  {
+    EmptyAwaitingData: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/patterns/empty-awaiting-data.png"}
+    ) {
+      childImageSharp {
+        gatsbyImageData(layout: FULL_WIDTH)
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/experiences/console/alarm-setup"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  description={props.pageContext.frontmatter.description}
+  categoryRoute={SidebarCategoryRoutes.EXPERIENCES}
+/>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<content>
+
+<WatchThisSpace>
+  <GatsbyImage image={props.data.EmptyAwaitingData.childImageSharp.gatsbyImageData} alt="" />
```

Comment:
nit: these images are showing up a little fuzzy for me, not sure whether there's an easy fix